libexec/azhpc-build.sh

```diff
@@ -205,8 +206,24 @@ for resource_name in $(jq -r ".resources | keys | @tsv" $config_file); do
             read_value resource_subnet ".resources.$resource_name.subnet"
             read_value resource_an ".resources.$resource_name.accelerated_networking" false
             read_value resource_instances ".resources.$resource_name.instances"
+            resource_disk_count=$(jq -r ".resources.$resource_name.data_disks | length" $config_file)
             resource_subnet_id="/subscriptions/$subscription_id/resourceGroups/$vnet_resource_group/providers/Microsoft.Network/virtualNetworks/$vnet_name/subnets/$resource_subnet"
 
+            resource_storage_sku=StandardSSD_LRS
+            data_disks_options=
+            if [ "$resource_disk_count" -gt 0 ]; then
+                read_value resource_storage_sku ".resources.$resource_name.storage_sku"
+                data_cache="ReadWrite"
+                resource_disk_sizes=$(jq -r ".resources.$resource_name.data_disks | @sh" $config_file)
+                for size in $resource_disk_sizes; do
+                    if [ $size -gt 4095 ]; then
+                        data_cache="None"
+                    fi
+                done
+                data_disks_options="--storage-sku $resource_storage_sku --data-disk-sizes-gb "$resource_disk_sizes" --data-disk-caching $data_cache "
```

Comment:
can we use the official term instead of storage sku - Storage Account Type?

Reply:
I used storage sku because that's the argument that the az vmss create requires (--storage-sku)

Reply:
ok lets keep it as such then